.github/workflows/ coderabbit.yml

```
@@ -0,0 +1,18 @@
+    branches: [main, develop]
+
+jobs:
+  review:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run CodeRabbit review
+      run: |
+        curl -L https://coderabbit.ai/api/run-review \
+        --header "Authorization: Bearer cr-7bf6531acd73250d533565e76c727636e4aa7d914ca52f8914af8408e5" \
+        --data "repository=${{ github.repository }}&commit=${{ github.sha }}"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)